code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -107,57 +107,73 @@
 		return
 	return ..()
 
-/obj/item/weapon/melee/baton/apply_hit_effect(mob/living/target, mob/living/user, var/hit_zone)
```

Comment:
Can anyone think of anything bad that could result from not passing in the hit zone here?

I don't like the idea of having to retrieve it from the user within this function, it makes overriding more of a pain.
Is it removed upstream or just in this override?